python/tvm/testing/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin, wildcard-import
+"""Utility Python functions for TVM testing"""
+from .utils import *
+from .auto_scheduler import *
```

Comment:
I am not in favor of wildcard import here, although it is in the testing folder :-) Let's import only when necessary here

Reply:
let's use `from . import auto_scheduler` instead

Reply:
As using `__init__.py` in `testing` will confuse the references to `tvm.testing`, I am removing it from this pr and opening up a new pr that contains this change.

Reply:
`__init__.py` is needed, and `testing.py` should be removed, otherwise it will shadow `tvm.testing` package

Reply:
done